2019-08-13 23:28:11 CarinaListener [INFO] 
============= Test configuration =============
url=http://qa.walook.com.mx:8081/#/login
env=DEMO
env_arg_resolver=com.qaprosoft.carina.core.foundation.utils.DefaultEnvArgResolver
platform=*
browser=chrome
browser_version=
browser_language=
selenium_host=http://localhost:4455/wd/hub
driver_event_listeners=
max_driver_count=3
custom_capabilities=
app_version=
proxy_host=
proxy_port=
proxy_protocols=http,https,ftp
browsermob_proxy=false
browsermob_host=
browsermob_port=0
browsermob_disabled_mitm=false
proxy_set_to_system=false
report_url=
email_list=
failure_email_list=
track_known_issues=false
auto_screenshot=true
smart_screenshot=false
implicit_timeout=10
explicit_timeout=20
auto_download=false
auto_download_apps=
custom_artifacts_folder=
retry_interval=5
project_report_directory=./reports/qa
max_screen_history=10
max_log_file_size=150
result_sorting=true
big_screen_width=-1
big_screen_height=-1
small_screen_width=90
small_screen_height=60
init_retry_count=0
init_retry_interval=1
retry_count=0
enable_l10n=true
l10n_encoding=ISO-8859-1
locale=en_US
enable_i18n=true
language=en_US
thread_count=1
data_provider_thread_count=-1
core_log_level=INFO
core_log_packages=com.qaprosoft.carina.core
artifacts_expiration_seconds=2592000
log_all_json=true
date_format=HH:mm:ss yyyy-MM-dd
time_format=HH:mm:ss
suite_name=
jira_updater=com.qaprosoft.carina.core.foundation.jira.DefaultJiraUpdater
jira_url=
jira_user=
jira_password=
jira_suite_id=
jira_project=
jira_project_short=
jira_create_new_ticket=false
test_naming_pattern={method_name}
optimize_video_recording=false
testrail_run_name=
testrail_milestone=
testrail_assignee=
qtest_cycle_name=DailyAutomation
qtest_suite_name=
s3_bucket_name=
access_key_id=
secret_key=
s3_local_storage=.
hockeyapp_token=
hockeyapp_local_storage=.
add_new_localization=false
add_new_localization_encoding=utf-8
add_new_localization_path=
add_new_localization_property_name=new_custom_messages_
tls_keysecure_location=
health_check_class=
health_check_methods=
uninstall_related_apps=false
default_device_timezone=GMT
default_device_time_format=24
default_device_language=en_US
android_screen_record_size=
android_screen_record_bitrate=
android_enable_bug_report=true
ios_screen_record_quality=MEDIUM
ios_screen_record_codec=libx264
ios_screen_record_fps=10
screen_record_duration=180
video_scale=-2:720
ignore_ssl=false
test_run_rules=
hub_mode=default
extract_sys_log=false
selenium_host=http://localhost:4455/wd/hub

------------- Driver capabilities -----------
================================================

2019-08-13 23:28:11 CarinaListener [ERROR] L10N bundle is not initialized successfully!
java.lang.ClassCastException: class jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to class java.net.URLClassLoader (jdk.internal.loader.ClassLoaders$AppClassLoader and java.net.URLClassLoader are in module java.base of loader 'bootstrap')
	at com.qaprosoft.carina.core.foundation.utils.resources.Resources.getResourceURLs(Resources.java:77)
	at com.qaprosoft.carina.core.foundation.utils.resources.L10N.init(L10N.java:59)
	at com.qaprosoft.carina.core.foundation.listeners.CarinaListener.<clinit>(CarinaListener.java:111)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.lang.Class.newInstance(Class.java:584)
	at org.testng.internal.ClassHelper.newInstance(ClassHelper.java:69)
	at org.testng.internal.DefaultListenerFactory.createListener(DefaultListenerFactory.java:13)
	at org.testng.TestRunner.initListeners(TestRunner.java:315)
	at org.testng.TestRunner.init(TestRunner.java:274)
	at org.testng.TestRunner.init(TestRunner.java:241)
	at org.testng.TestRunner.<init>(TestRunner.java:167)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:663)
	at org.testng.SuiteRunner.init(SuiteRunner.java:260)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:198)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1295)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1273)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2019-08-13 23:28:11 CarinaListener [ERROR] I18N bundle is not initialized successfully!
java.lang.ClassCastException: class jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to class java.net.URLClassLoader (jdk.internal.loader.ClassLoaders$AppClassLoader and java.net.URLClassLoader are in module java.base of loader 'bootstrap')
	at com.qaprosoft.carina.core.foundation.utils.resources.Resources.getResourceURLs(Resources.java:77)
	at com.qaprosoft.carina.core.foundation.utils.resources.I18N.init(I18N.java:56)
	at com.qaprosoft.carina.core.foundation.listeners.CarinaListener.<clinit>(CarinaListener.java:117)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.lang.Class.newInstance(Class.java:584)
	at org.testng.internal.ClassHelper.newInstance(ClassHelper.java:69)
	at org.testng.internal.DefaultListenerFactory.createListener(DefaultListenerFactory.java:13)
	at org.testng.TestRunner.initListeners(TestRunner.java:315)
	at org.testng.TestRunner.init(TestRunner.java:274)
	at org.testng.TestRunner.init(TestRunner.java:241)
	at org.testng.TestRunner.<init>(TestRunner.java:167)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:663)
	at org.testng.SuiteRunner.init(SuiteRunner.java:260)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:198)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1295)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1273)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2019-08-13 23:28:11 Messager [INFO] INFO:  TEST [Forte Demo - testLoginSucces] STARTED at [23:28:11 2019-08-13]
2019-08-13 23:28:12 IDriverPool [ERROR] Driver initialization 'default' FAILED! Retry 1 of 1 time - Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'sanpsicro', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.18.0-25-generic', java.version: '11.0.3'
Driver info: driver.version: RemoteWebDriver
org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'sanpsicro', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.18.0-25-generic', java.version: '11.0.3'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at com.qaprosoft.carina.core.foundation.webdriver.core.factory.impl.DesktopFactory.create(DesktopFactory.java:87)
	at com.qaprosoft.carina.core.foundation.webdriver.core.factory.DriverFactory.create(DriverFactory.java:71)
	at com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.createDriver(IDriverPool.java:351)
	at com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:143)
	at com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:78)
	at com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:66)
	at com.test.forte.demo.Registro_test.testLoginSucces(Registro_test.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)
	at com.qaprosoft.zafira.listener.ZafiraListener.lambda$run$0(ZafiraListener.java:102)
	at com.qaprosoft.zafira.listener.ZafiraEventRegistrar.onTestHook(ZafiraEventRegistrar.java:339)
	at com.qaprosoft.zafira.listener.ZafiraListener.run(ZafiraListener.java:102)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:579)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:4455
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
	at okhttp3.RealCall.execute(RealCall.java:77)
	at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at com.qaprosoft.carina.core.foundation.webdriver.listener.EventFiringSeleniumCommandExecutor.execute(EventFiringSeleniumCommandExecutor.java:49)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	... 34 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:591)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
	... 57 more
2019-08-13 23:28:13 Messager [INFO] INFO:  TEST [Forte Demo - testLoginSucces] FAILED at [23:28:13 2019-08-13] - org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'sanpsicro', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.18.0-25-generic', java.version: '11.0.3'
Driver info: driver.version: RemoteWebDriver

com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.createDriver(IDriverPool.java:410)
com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:143)
com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:78)
com.qaprosoft.carina.core.foundation.webdriver.IDriverPool.getDriver(IDriverPool.java:66)
com.test.forte.demo.Registro_test.testLoginSucces(Registro_test.java:54)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.base/java.lang.reflect.Method.invoke(Method.java:566)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)
com.qaprosoft.zafira.listener.ZafiraListener.lambda$run$0(ZafiraListener.java:102)
com.qaprosoft.zafira.listener.ZafiraEventRegistrar.onTestHook(ZafiraEventRegistrar.java:339)
com.qaprosoft.zafira.listener.ZafiraListener.run(ZafiraListener.java:102)
org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)
org.testng.internal.Invoker.invokeMethod(Invoker.java:579)
org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
org.testng.TestRunner.privateRun(TestRunner.java:648)
org.testng.TestRunner.run(TestRunner.java:505)
org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
java.base/java.lang.Thread.run(Thread.java:834)
2019-08-13 23:28:13 Messager [INFO] INFO: '**************** Test execution summary ****************'.
2019-08-13 23:28:13 Messager [INFO] RESULT #1: TEST [Forte Demo - testLoginSucces] FAIL [log=file:///home/sanpsicro/Desktop/Frameworks/Carina/automation/./reports/qa/1565756891234/Forte_Demo_-_testLoginSucces/test.log]
